### PR TITLE
updated wrapper to build kernel librairies

### PR DESCRIPTION
**Pull request by QuentinOchem**

### CODE REVIEW

File: examples/0_Simple/vectorAdd_lib/host.gpr [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+   for Exec_Dir use ".";
+   for Object_Dir use "obj/host";
+   for Source_Dirs use ("src/common", "src/host");
+   for Main use ("main.adb");
+
+   for Create_Missing_Dirs use "True";
+
+   package Compiler is
+      for Switches ("ada") use ("-gnatX", "-g", "-gnatd_c", "-gnatDGL");
```
Comment (glacambre) [inline examples/0_Simple/vectorAdd_lib/host.gpr:27]: Argh, this is a leftover from my examples and isn't actually needed (`-gnatDGL` makes GNAT emit its expansion code, which is useful for debugging GNAT but not much else).
```suggestion
      for Switches ("ada") use ("-gnatX", "-g", "-gnatd_c");
```

File: examples/0_Simple/vectorAdd_lib/host.gpr [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+   end Compiler;
+
+   package Linker is
+      for Switches ("ada") use ("-L/usr/lib/cuda/targets/x86_64-linux/lib/stubs", "-L/usr/lib/cuda/targets/x86_64-linux/lib", "-lcudadevrt", "-lcudart_static", "-lrt", "-lpthread", "-ldl",
+         "-Wl,--unresolved-symbols=ignore-all");
```
Comment (glacambre) [inline examples/0_Simple/vectorAdd_lib/host.gpr:32]: I don't think `-Wl,--unresolved-symbols=ignore-all` is needed anymore, and if it is there's a problem :/
```suggestion
      for Switches ("ada") use ("-L/usr/lib/cuda/targets/x86_64-linux/lib/stubs", "-L/usr/lib/cuda/targets/x86_64-linux/lib", "-lcudadevrt", "-lcudart_static", "-lrt", "-lpthread", "-ldl");
```

**Review by glacambre — APPROVED** [review]
LGTM!

**Review by ohenley — APPROVED** [review]
Its OK.

**Pull request merged**